openapi/openapi.go

```diff
@@ -74,17 +76,21 @@ func LoadFromYaml(openApiFilepath string) (*OpenAPI, error) {
 	return &api, err
 }
 
+// ResolveReferences traverses the entire openapi struct tree and loads te h
+// TODO: this doesn't have to be public
 func (o *OpenAPI) ResolveReferences(absoluteBasePath string) error {
 	o.SetBasePath(filepath.Dir(absoluteBasePath))
+	// TODO: what if JSON?
```

Comment:
Why does the input format have any implication to this function?

Reply:
iirc it's because of the use of tree.ResolveRefs